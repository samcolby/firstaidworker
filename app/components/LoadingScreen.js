import React from "react";
import { ActivityIndicator, StatusBar, StyleSheet, View } from "react-native";

import { COLOURS } from "../Constants";

function LoadingScreen() {
  return (
    <View>
      <StatusBar networkActivityIndicatorVisible={true} />
      <ActivityIndicator
        style={styles.activityIndicator}
        size="large"
        color={COLOURS.TAB_HINTS}
      />
    </View>
  );
}

var styles = StyleSheet.create({
  activityIndicator: {
    marginTop: 20
  }
});

export default LoadingScreen;
