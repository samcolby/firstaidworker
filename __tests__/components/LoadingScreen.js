import React from "react";

import { shallow } from "enzyme";

import LoadingScreen from "../../app/components/LoadingScreen";

describe("LoadingScreen Component", () => {
  it("should render without issues", () => {
    const component = shallow(<LoadingScreen />);

    expect(component.length).toBe(1);
    expect(component).toMatchSnapshot();
  });
});
