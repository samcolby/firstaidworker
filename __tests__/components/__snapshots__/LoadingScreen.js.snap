// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen Component should render without issues 1`] = `
<View>
  <StatusBar
    animated={false}
    networkActivityIndicatorVisible={true}
    showHideTransition="fade"
  />
  <ActivityIndicator
    animating={true}
    color="red"
    hidesWhenStopped={true}
    size="large"
    style={
      Object {
        "marginTop": 20,
      }
    }
  />
</View>
`;
